README.md

```diff
@@ -24,16 +32,11 @@ $ docker run --restart=always \
 	--name=shifthelper_mysql \
 	mysql
 ```
-Start the shifthelper container
+
+Adapt the database settings in the config file to reflect your local db setup,
+then run
```

Comment:
you mean the MYSQL_USER and MYSQL_PASSWORD from above right? 
So alternatively one could use the values from the webservice.json in the `docker run mysql` call and avoid touching the config file, yes?